build/win32/Cargo.toml

```diff
@@ -1,19 +0,0 @@
-[package]
```

Comment:
Not entirely sure what this file is all about, but it kept showing up as a local change.